src/components/math-quotes.js

```diff
@@ -0,0 +1,50 @@
+import { useEffect, useState } from 'react';
+
+const url = 'https://api.api-ninjas.com/v1/quotes?category=computers';
+const API_KEY = 'En6KYhKgbYaCLmYOVtqFhA==W6viOXJ3erlnK3aa';
+
+const ShowRandomMathQuote = () => {
+  const [isLoading, setIsLoading] = useState(true);
+  const [isError, setIsError] = useState(false);
+  const [data, setData] = useState(null);
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const response = await fetch(url, {
+          headers: { 'X-Api-Key': API_KEY },
+          contentType: 'application/json',
+        });
+        const data = await response.json();
+        setData(data);
+      } catch (err) {
+        setIsError(true);
+        console.log(err);
```

Comment:
- [ ] [OPTIONAL] Using `console.log` in client-side apps may lead to some security risks, one of which is sensitive information may be leaked. I highly suggest you remove it after you finished debugging your app. 😎️

Reply:
Hello 👋 @amejid,here in are the implemented changes

Reply:
This change has been done well